descriptors/ModuleDescriptor-template.json

```diff
@@ -460,12 +460,12 @@
   ],
   "permissionSets": [
     {
-      "permissionName": "mod-settings.global.write.authority-storage",
+      "permissionName": "mod-settings.global.write.authority-storage.manage",
       "displayName": "Entities links settings - Create configuration",
       "description": "To create new configuration in mod settings"
     },
     {
-      "permissionName": "mod-settings.global.read.authority-storage",
+      "permissionName": "mod-settings.global.read.authority-storage.manage",
```

Comment:
You should add "replace" with old permission. It will help with smooth migration of user permissions.